client/packages/coldchain/src/Equipment/utils.ts

```diff
@@ -170,6 +166,7 @@ export const importEquipmentToCsv = (
   fields.push(...baseAssetFields(t), ...dedupedAssetProperties);
 
   const data = assets.map(node => {
+    const parsedStatus = parseLogStatus(node.status??StatusType.Functioning)?.key // to avoid status being undefined
```

Comment:
Would probably call this something like `statusKey`, rather than `parsedStatus` (the parsed status would be what is returned from `parseLogStatus`, but you're now accessing the `key` property here 🙏)